spec.md

```diff
@@ -693,15 +723,15 @@ Procedure:
 
 7. if X1 != X2 return INVALID
 
-8. c = H(Abar || A' || h0 || C1 || d || h0 || h\_i || ... || h\_R || C2 || nonce)
+8. c = H(Abar || A' || h0 || C1 || d || h0 || h\[i\] || ... || h\[R\] || C2 || nonce)
 
 9. T1 = Abar - d
 
-10. T2 = P1 + h\_i \* -m^\_i + ... + h\_D \* -m^\_D
+10. T2 = P1 + h\[i\] \* msg\[i\] + ... + h\[D\] \* msg\[D\]
```

Comment:
Updated the formula here, and in item 12. The previous version added `P1*c` to `Y2` instead of subtracting it.